repos/spack_repo/builtin/packages/mochi_margo/package.py

```diff
@@ -18,6 +19,8 @@ class MochiMargo(AutotoolsPackage):
     maintainers("carns", "mdorier", "fbudin69500")
 
     version("main", branch="main")
```

Comment:
```suggestion
    build_system("cmake", "autotools", default="CMake")

    version("main", branch="main")
```
with `conditional` as appropriate.

Reply:
Thanks, I added. Now all tests are passing.